Week03/lesson06/classwork/startup_matchmaker/css/style.css

```diff
@@ -0,0 +1,169 @@
+* {
+     -moz-box-sizing: border-box; /* Firexfox */
+     -webkit-box-sizing: border-box; /* Safari/Chrome/iOS/Android */
+     box-sizing: border-box; /* IE */
+}
+
+/* Clear fix hack */
+.clearfix:before, .clearfix:after {
+  content: "";
+  display: table;
+}
+
+.clearfix:after{
+  clear: both;
+}
+
+body {
+     font-size: 25px;
+     font-family: Source Sans Pro, Arial;
+     color: #000033;
+}
+
+header {
+       height: 210px;
+       background-color: #000000;
+}
+
+h1 {
+    font-size: 40px;
+    font-family: merriweather;
+    color: #ffff00;
+    padding: 55px 0 50px 40px;
+    display: inline-block;
+}
+
+a {
+  text-decoration: none;
+}
+
+a:hover {
+        color: #ffff00;
+}
+
+a:active {
+         background-color: #808080;
+         border-radius: 5px;
+}
+
+nav a {
+      color: #ffffff;
+      padding-right: 30px;
+}
+
+nav {
+    padding: 60px 40px 0 0;
+    float:right;
+}
+
+.pages {
+       font-size: 30px;
+       font-family: Oswald;
+       text-transform: capitalize;
+       display:inline-block;
+}
+
+header p {
+         border-top:5px solid #ffff00;
+         font-style:italic;
+         text-align: center;
+         background-color: #808080;
+         padding: 15px 0 20px 0;
+}
+
+span {
+     text-decoration: line-through;
+     color: #bfbfbf;
+}
+
+em {
+   text-decoration: underline;
+   text-decoration-color: #ffff00
+}
+
+main{
+    background: url(../images/coworking.jpg) no-repeat;
+    background-size: cover;
+    height: 600px;
+}
+
+.boxright {
+          height: 500px;
+          width: 400px;
+          background: rgba(255, 255, 255, 0.5);
+          line-height: 1.5em;
+          padding: 20px 40px 20px 40px;
+          margin: 50px 20px 50px 0;
+          border: 2px solid #000000;
+          float:right;
+}
+
+.boxright p {
+            padding-bottom: 20px;
+}
+
+h2 {
+   font-size: 30px;
+   font-family: Oswald;
+   line-height: 2em;
+ }
+
+ .link {
+       height: 65px;
+       width: 250px;
+       line-height: 2.5em;
+       background-color: #808080;
+       margin: 0 auto;
+       border-radius: 5px;
+ }
+
+.boxright .link {
+                background-color: #ffff00;
+                padding: 0 0 0 35px;
+}
+
+.boxright .link a:hover {
+              color: #808080;
+}
+
+.link a {
+        font-family: Oswald;
+        text-transform: capitalize;
+        text-align: center;
+        color: #000033;
+        display:inline-block;
+}
+
+section {
+          height: 350px;
+          padding-left: 100px;
+          padding-top: 50px;
```

Comment:
Your use of padding here makes me think that maybe you're having trouble centering things on the page. The key to centering a block is to limit its width and then `auto` its left and right margins, like so:

```
section {
  width: 1000px;
  margin: 50px auto; /* 50px top and bottom, auto left and right */
}
```

And now, in order to get the three `.box` elements sitting nicely side by side, you can do the maths:

320 * 3 + 20 + 20 = 1000 (that's three boxes that are 320px wide with two 20px gutters in between)

So given this HTML structure:

![image](https://cloud.githubusercontent.com/assets/5670279/17491706/9cf03cdc-5da0-11e6-95d9-a1bb91562429.png)
(By the way you missed a great opportunity to let `.box` to the heavy-lifting of the styling for that second box and just let the `.middlebox` class take care of the borders and nothing else. But good call on applying borders just to middle box!)

... so assuming that HTML structure, you'd want some CSS that looks like this:

```
section {
  width: 1000px;
  margin: 50px auto;
  font-size: 0; /* remember the "font-size: 0" hack? This is the perfect place to use it! Without it here the hidden newline characters (the character that gets encoded when you hit the "Enter" key is adding extra pixels between your boxes, so the maths above wouldn't work, though they should */
}

.box {
  width: 320px;
  padding: 20px;
  display: inline-block;
  margin-right: 20px;
  font-size: 25px;
  vertical-align: top; /* This is what ensures that even if all the boxes aren't the same height (based on their content) that the tops of the boxes will align */
}

.middlebox {
  border-right: 2px solid #808080;
  border-left: 2px solid #808080;
}
```